Public/Src/Sandbox/Windows/DetoursServices/SendReport.cpp

```diff
@@ -81,7 +81,11 @@ void ReportFileAccess(
     else {
         fileName = policyResult.GetCanonicalizedPath().GetPathString();
     }
-
+    
+    if (wcspbrk((wchar_t*)fileName, L"\u000A\u000B\u000C\u000D\u0085\u2028\u2029")) {
+        fileName = L"";

```

Comment:
This will cause the bxl engine to ignore this file access since we're ignoring the rest of the filename if a newline character is found by always setting the filename to empty.
https://github.com/microsoft/BuildXL/blob/4663518d2d9afb8ae9884b768816b596054ce176/Public/Src/Engine/Processes/SandboxedProcessReports.cs#L692
This is probably not the behaviour you want.

Reply:
I think this is precisely what we want. In case a filename is broken by having a newline character, we would ideally want to skip it, as any attempt to process it through ReportAccesses leads to incorrect parsing of the filename.